modules/nodes-descheduler-installing.adoc

```diff
@@ -64,7 +64,31 @@ ifdef::nodes[]
 ====
 Do not enable both `TopologyAndDuplicates` and `SoftTopologyAndDuplicates`. Enabling both results in a conflict.
 ====
-... Optional: Expand the *Profile Customizations* section to set a custom pod lifetime value for the `LifecycleAndUtilization` profile. Valid units are `s`, `m`, or `h`. The default pod lifetime is 24 hours.
+... Optional: Expand the *Profile Customizations* section to set optional configurations for the descheduler.
+**** Set a custom pod lifetime value for the `LifecycleAndUtilization` profile. Use the *podLifetime* field to set a numerical value and a valid unit (`s`, `m`, or `h`). The default pod lifetime is 24 hours.
```

Comment:
Would it be helpful to show how this value looks? Maybe for the 24 hours value? Not sure if it's `24h`, `24 h`, or something else. If there's text validation in the UI, an example is probably superfluous but I don't have a cluster up right now to look :slightly_smiling_face: